exercises/practice/accumulate/.meta/config.json

```diff
@@ -3,8 +3,8 @@
   "authors": [],
   "files": {
     "solution": [],
-    "test": [],
-    "example": []
+    "test": ["runtests.jl"],
+    "example": [".meta/example.jl"]
```

Comment:
Is this just part of the spec now?

Reply:
Yes, see https://github.com/exercism/docs/blob/main/anatomy/tracks/practice-exercises.md#file-metaconfigjson

The reason it's in this PR is that I originally wanted `runtests.jl` to use the file paths from the config, but then afterwards figured instead of introducing a dependency, I might as well hardcode them since they're identical across exercises.